doc/builders.rst

```diff
@@ -341,6 +341,18 @@ for details.
 
    .. autoattribute:: supported_image_types
 
+.. module:: sphinx.builders.dummy
+.. class:: DummyBuilder
+
+   This builder produces no output.  The input is only parsed and checked for
+   consistency.  This is useful for linting purposes.
+
+   .. autoattribute:: name
+
+   .. autoattribute:: supported_image_types
```

Comment:
In the page, almost of builders have `supported_image_types` entry if the builder supports no image format
http://www.sphinx-doc.org/en/stable/builders.html#sphinx.builders.text.TextBuilder

I'd rather adding `format` attribute than removing it.